README.md

```diff
@@ -18,19 +18,72 @@ We are going to track some new features/additions here so that it is quick and e
 - we've renamed the distributed .env files to ```switchboard.env.dist``` and ```frontend.env.dist```. This ensures that your local
   configuration doesn't get blown away when you pull changes from the repo. (We still use ```switchboard.env``` and ```frontend.env```
   for the config, it just means that new clones of the repo require the users to copy / rename the dist files)
-
 - we have added an extra `switchboard.env` called `CANARY_IPINFO_API_KEY`. This allows you to use your ipinfo.io api key if you
   want to (keep in mind ipinfo.io does have a free tier of up to 1000 requests a day).
-
 - we now have slack support. When you supply a webhook, you simply supply your slack webhook url. (Thanks to @shortstack).
-
 - we have added a new environment variable to `frontend.env` called `CANARY_AWSID_URL` which allows you to specify a private or
   different url for the AWS ID token. This means you can easily change between accounts. (2018-10-17)
 
+
+
+## Domain name / DNS configuration
+
+You register a domain name using a "registrar". This "registrar" has a communication channel with the "registry" which is the entity managing the TLD (`.com` for example).
+
+The registry operate some DNS servers that list domain names available into it's TLD, with the corresponding "next" DNS servers in the chain to send requests to for further records inside this domain zone.
+
+You configure in admin panel of you registrar, what DNS servers you want the registrar to "push" into the DNS servers managed by the registry.
+
+
+
+- Registry DNS level
+
+This information is defined into the Registrar admin panel (then pushed to Registry DNS servers)
+
+```
+example.com.    NS    ns1.my.dns.servers.
+example.com .   NS    ns2.my.dns.servers.
+```
+
+
+
+- Domain DNS level
+
+On `nsX.my.dns.servers.`  DNS servers, configure the following example of zone content
```

Comment:
Since we removing the above we'll need to reword this a little. We want to make this as straight forward for folks to follow. So maybe something like:

Once you have your domain name `example.com`, you will need to configure your DNS zone content. If we have our Canarytokens server running at IP `1.2.3.4`, our DNS zone configuration would look something like

Reply:
:+1: